src/plugins/data_view_editor/public/components/form_schema.ts

```diff
@@ -28,9 +28,9 @@ export const schema = {
       defaultMessage: 'Index pattern',
     }),
     defaultValue: '',
-    helpText: i18n.translate('indexPatternEditor.validations.titleHelpText', {
+    helpText: i18n.translate('indexPatternEditor.validations.titleInputHelpText', {
       defaultMessage:
-        'Enter an index pattern that matches one or more data sources. Use an asterisk (*) to match multiple characters. Spaces and the characters , /, ?, ", <, >, | are not allowed.',
+        'Enter an index pattern that matches one or more data sources. Use a wildcard (*) to match multiple sources. Separate with a comma (,) to match multiple single sources. Spaces and the characters /?"<>| are not allowed.',
```

Comment:
@gchaps And this change.

<img width="600" alt="Screenshot 2023-03-01 at 12 02 40" src="https://user-images.githubusercontent.com/1415710/222222402-526201c1-d2b4-4974-91a3-894a5827f5f3.png">